oapi-codegen.config.yaml

```diff
@@ -0,0 +1,5 @@
+output: pkg/openapi-client/wego_server.go
+package: wego_client
+generate:
+  - types
+  - client
```

Comment:
Neat! I only had a very quick play w/ this and didn't work immediately, any luck? swagger-codegen was the other I've seen around in grpc-gateway examples etc.